CLOUDP-128165: Create command "atlas backups schedule describe" (#1360)
mongodb/mongodb-atlas-cli: docs/atlascli/command/atlas-backups.txt

* :ref:`atlas-backups-exports` - Manage cloud backup export jobs for your project.
* :ref:`atlas-backups-restores` - Manage cloud backup restore jobs for your project.

> * :ref:`atlas-backups-schedule` - Get cloud backup schedule of a given cluster in your project.

Reviewer: ```suggestion
* :ref:`atlas-backups-schedule` - Return a cloud backup schedule for the cluster you specify.
```
